README.md

```diff
@@ -190,6 +190,12 @@ the `known_hosts` file is provided.  This can help avoid issues for hosts with
 dynamic IP addresses, but removes some additional protection against DNS
 spoofing attacks.  Host IP Checking is enabled by default.
 
+#### ENV_SSH_KEY
```

Comment:
Standard convention to override `VARIABLE_FILE` is to use `VARIABLE`, not `ENV_VARIABLE`.